pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -615,3 +621,30 @@ func GetNodeAddr(node *corev1.Node) (net.IP, error) {
 	}
 	return ipAddr, nil
 }
+
+func (c *Controller) IPInPodSubnets(ip net.IP) bool {
```

Comment:
Could we add a simple unit test for this method?

Reply:
Sure, added.